### ISSUE TITLE
CLI to parse into csv files

### ISSUE THREAD

**Issue by joweich** [joweich/chat-miner#105]
## Goal
Add a command line interface that allows for parsing chat export files into csv files.
In its minimal version, the usage would like like this:

```
chatminer --parser whatsapp --input chatlog.txt --output export.csv
```

## Jobs to be done:
- [ ] Add new file `cli.py`
- [ ] Add logic parsing and checking command line arguments
- [ ] Trigger chat file parsing and csv export of resulting dataframe
- [ ] [OPTIONAL] Add output for the `--help` flag explaining the usage of the CLI

**Comment by Exterminator11** [comment]
Hey @joweich , could I work on this issue

**Comment by joweich** [comment]
@Exterminator11  sure, I'm happy to assign this issue to you! 🙂

**Comment by Exterminator11** [comment]
Hey @joweich, I was able to come up with a `cli.py` file and just to ensure I understood the issue correctly, do these screenshots match the functionality you are looking for?

<img width="1141" alt="Screenshot 2023-09-10 at 7 52 41 AM" src="https://github.com/joweich/chat-miner/assets/88940647/7f592d17-647c-4f38-81f7-247cce0557d4">
<img width="1111" alt="Screenshot 2023-09-10 at 7 51 36 AM" src="https://github.com/joweich/chat-miner/assets/88940647/661d8d9a-6fe7-4034-ad07-a9e94cbc6c8d">

At this point I am executing the file using the complete path I am yet to change that to match `chatminer --parser whatsapp --input chatlog.txt --output export.csv`

**Comment by joweich** [comment]
@Exterminator11 yes, that's perfect!

**Comment by Exterminator11** [comment]
Hey @joweich, 
I was able to create the command line for it, this is what it looks like.
<img width="1129" alt="Screenshot 2023-09-10 at 8 40 03 PM" src="https://github.com/joweich/chat-miner/assets/88940647/4501c7bb-c68e-41d6-99b7-40789198ce72">
<img width="1141" alt="Screenshot 2023-09-10 at 8 40 15 PM" src="https://github.com/joweich/chat-miner/assets/88940647/94ab192b-60bf-4d8e-b808-9a45eea9330d">
I am a newcomer to open-source contributions 😅, and I would greatly appreciate your guidance on the subsequent steps.

**Comment by joweich** [comment]
@Exterminator11 happy to! You'll first commit and push your code changes to your fork of this project. After, you'll open a Pull Request, which I will subsequently merge to the main branch of this project

**Comment by Exterminator11** [comment]
Alright, thank you so much for the support:)

**Comment by joweich** [comment]
Closed via #106
